hack/generate/samples/internal/go/memcached-with-webhooks/memcached_with_webhooks.go

```diff
@@ -652,7 +741,7 @@ func getPodNames(pods []corev1.Pod) []string {
 }
 `
 
-const importsFragment = `
+const contorollerImportsFragment = `
```

Comment:
Nit - same as above:
```suggestion
const controllerImportsFragment = `
```

Reply:
Done